rero_ils/modules/users/api.py

```diff
@@ -59,6 +59,8 @@ def get_readonly_profile_fields():
 class User(object):
     """User API."""
 
+    _default_schema = 'https://bib.rero.ch/schemas/users/user-v0.0.1.json'
```

Comment:
Any reason, why this is not in `config.py` like the other resources => `RECORDS_JSON_SCHEM`?

Reply:
because this isn't a real Invenio resource.
But you're right, I will create a function to build this URL based on values from config
👍

Reply:
You can access the config from the console without any problem.